packages/dso-toolkit/components/Voorbeelden/20-Toepassingen/40-Mijn-Omgevingsloket/Project overzicht.njk

```diff
@@ -11,6 +11,29 @@
     </div>
   </div>
   <div class="row">
+    <div class="col-md-4">
+      <div class="row dso-equal-heights">
+        <div class="col-md-12 col-sm-6">
+          <nav aria-label="projecttaken">
```

Comment:
Klopt dit?

We hebben afgesproken dat Link List verantwoordelijk is voor het maken van 'n `<nav />`, zoals we in de template functies hebben gemaakt: https://github.com/dso-toolkit/dso-toolkit/pull/1473/files#diff-94d5bb107a585116bf4ad752aff4e68db7aa1a80135738d5618f0b73f53c43ccR38

Hier ontstaat er zomaar ineens 'n nav. Is de situatie niet te vergelijken of lopen de routines uit de pas. Ik verwacht namelijk dat `link-list.njk` dezelfde wijzigingen heeft die in `link-list.template.ts` zijn gerealiseerd.

Reply:
Je hebt helemaal gelijk, dit liep nog uit de pas